wp-content/themes/defense360/inc/allowed-blocks.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/** 
+ * Defines allowed blocks to be used in theme.
+ * 
+ * Source: https://rudrastyh.com/gutenberg/remove-default-blocks.html#block_slugs
+*/
+
+add_filter( 'allowed_block_types', 'defense360_allowed_block_types');
+ 
+function defense360_allowed_block_types( $allowed_blocks) {
+ 
+	return array(
+		'core/image',
+		'core/paragraph',
+        'core/heading',
+        'core/list',
+        'core/quote',
+        // We may include either regular quotes or pull quotes.
+        // 'core/pullquote',
+        'core/shortcode',
+        'core/gallery',
+        'core/freeform',
+
+    );
+}
+?>
```

Comment:
Can you add a blank line at the end of the file? You also don't need the closing `?>`

Reply:
I went ahead and added the additional blocks and reordered them alphabetically. Also fixed indentation.

As far as plugins being affected. It looks like the whitelist does affect those. I was trying to find what the jetpack slugs were called so I could add them to the list, but I haven't had any luck.